src/css/mines.scss

```diff
@@ -0,0 +1,167 @@
+.cell {
```

Comment:
What advantages gives u scss here?

Reply:
at least, nested selectors

Reply:
what for? u can do it with basic css. Why did u choose .scss?

Reply:
OK, It's not a mistake, but keep in mind that it can be tricky